Sortie en erreur de la CI lors du merge
```
  Campagne de collecte de profils
    ✓ Je partage mon profil (5998ms)
    ✓ Je partage mon profil via l'URL sans être connecté (4847ms)
    ✓ Je partage mon profil de manière restreinte (5945ms)
    1) Je partage mon profil de manière restreinte en étant connecté via un organisme externe


  3 passing (23s)
  1 failing

  1) Campagne de collecte de profils
       Je partage mon profil de manière restreinte en étant connecté via un organisme externe:
     AssertionError: Timed out retrying: expected 'http://localhost:4200/campagnes/WOLF/privee/identification' to include 'rejoindre'
```

--- a/high-level-tests/load-testing/README.md
+++ b/high-level-tests/load-testing/README.md
@@ -1,3 +1,30 @@
+# Génération des JDD
+
+## fichier SIECLE
+
+Déterminer l'UAJ de l'organisation (par défaut, 1237457A).
+
+```
+SELECT o.id, o.name, o."externalId" AS UAJ
+FROM organizations o
+WHERE o."isManagingStudents" = true ORDER BY o.id
+``` 
+
+Déterminer le volume à générer (par défaut, 5)  
+
+Exécuter `npm run generate-bulk-data:schooling-registrations <NOMBRE_UTILISATEURS> <UAI_ORGANISATION>`
+Exemple  
+- `npm run generate-bulk-data:schooling-registrations 4000 1237457A`
+- `npm run generate-bulk-data:schooling-registrations`
+
+Vous obtenez le message suivant et un fichier est généré (10 Mb pour 4000 utilisateurs)
+```
+Generating SIECLE file on organization 1237457A for 4000 users
+SIECLE file for 4000 users in file SIECLE-organization-1237457A-4000-users.xml
+```
+
+# Exécution des test
+
 ## Pré-requis :
 
 Faire tourner l'API en désactivant reCAPTCHA et MailJet. [PR #478](https://github.com/1024pix/pix/pull/478)

--- a/high-level-tests/load-testing/data/generate-schooling-registrations.js
+++ b/high-level-tests/load-testing/data/generate-schooling-registrations.js
@@ -1,0 +1,140 @@
+const js2xmlparser = require('js2xmlparser');
+const faker = require('faker');
+const fs = require('fs');
+
+const run = function() {
+
+  const DEFAULT_USER_COUNT = 5;
+  const userCount = process.argv[2] || DEFAULT_USER_COUNT;
+
+  const DEFAULT_ORGANIZATION_UAJ = '1237457A';
+  const organizationUAJ =  process.argv[3] || DEFAULT_ORGANIZATION_UAJ;
+
+  console.log(`Generating SIECLE file on organization ${organizationUAJ} for ${userCount} users`);
+
+  const PARAMETRES = {
+    UAJ: organizationUAJ,
+    ANNEE_SCOLAIRE: '2019',
+    DATE_EXPORT: '29/08/2019',
+    HORODATAGE: '29/08/2019 15:25:08',
+  };
+
+  const ELEVES = generateELEVES(userCount);
+
+  const STRUCTURES = generateSTRUCTURES(ELEVES);
+
+  const DONNEES = {
+    ELEVES: { ELEVE: ELEVES } ,
+    OPTIONS: {},
+    STRUCTURES: { STRUCTURES_ELEVE : STRUCTURES },
+    BOURSES: {},
+    ADRESSES: {},
+  };
+
+  const BEE_ELEVES = {
+    PARAMETRES,
+    DONNEES,
+  };
+
+  const rootXMLNode = 'BEE_ELEVES';
+  const SIECLEFileContent = js2xmlparser.parse(rootXMLNode, BEE_ELEVES);
+  const fileName = `SIECLE-organization-${organizationUAJ}-${userCount}-users.xml`;
+
+  fs.writeFile(fileName , SIECLEFileContent, function(err) {
+    if (err) throw err;
+    console.log(`SIECLE file for ${userCount} users in file ${fileName}`);
+  });
+
+};
+
+const generateELEVES = function(userCount) {
+
+  const ELEVES = [];
+
+  const identifier = function(base, length) {
+    const padWith = '0';
+    return base.toString().padStart(length, padWith);
+  };
+
+  for (let i = 0; i < userCount; i++) {
+
+    const ELEVE = {
+      '@': {
+        ELEVE_ID: identifier(i,7), // 7  '1223189',
+        ELENOET:  identifier(i,4),    //  4 '6202'
+      },
+      ID_NATIONAL: identifier(i,12), // 12: '123456789AB',
+      INE_BEA:     identifier(i,11), // 11: '1234567890A',
+      INE_RNIE:    identifier(i,12), //  12:'123456789AB',
+      ELENOET:     identifier(i,4), // See @
+      ID_ELEVE_ETAB: identifier(i,11), // '12345678901'
+      NOM_DE_FAMILLE: faker.name.lastName(),
+      PRENOM:  faker.name.firstName(),
+      PRENOM2: faker.name.firstName(),
+      PRENOM3: faker.name.firstName(),
+      DATE_NAISS: '01/03/2001',
+      DOUBLEMENT: '1',
+      CODE_PAYS: '100',
+      ACCEPTE_SMS: '0',
+      DATE_MODIFICATION: '27/08/2019',
+      CODE_REGIME: '0',
+      DATE_ENTREE: '01/09/2015',
+      VILLE_NAISS: '',
+      CODE_SEXE: '2',
+      CODE_PAYS_NAT: '100',
+      CODE_STATUT: 'ST',
+      CODE_MEF: identifier(i,11), // 11 '12345678901',
+      CODE_DEPARTEMENT_NAISS: '041',
+      CODE_COMMUNE_INSEE_NAISS: '41018',
+      ADHESION_TRANSPORT: '0',
+      CODE_PROVENANCE: '1',
+      SCOLARITE_AN_DERNIER: {
+        CODE_MEF: identifier(i,11), //11 '12345678901',
+        CODE_STRUCTURE: '3E 4',
+        CODE_RNE: '0180777X',
+        CODE_NATURE: '340',
+        SIGLE: 'CLG',
+        DENOM_PRINC: 'COLLEGE',
+        DENOM_COMPL: 'JEAN RENOIR',
+        LIGNE1_ADRESSE: '40 rue des Fileuses',
+        BOITE_POSTALE: '',
+        MEL: 'ce.0180777X@ac-orleans-tours.fr',
+        TELEPHONE: '0248202445',
+        CODE_COMMUNE_INSEE: '18033',
+        LL_COMMUNE_INSEE: 'BOURGES',
+      },
+    };
+
+    ELEVES.push(ELEVE);
+  }
+
+  return ELEVES;
+};
+
+const generateSTRUCTURES = function(ELEVES) {
+
+  const STRUCTURES = [];
+
+  ELEVES.forEach((eleve) => {
+
+    const STRUCTURES_ELEVE = {
+      '@': {
+        ELEVE_ID: eleve['@'].ELEVE_ID,
+        ELENOET: eleve['@'].ELENOET,
+      },
+      STRUCTURE: {
+        CODE_STRUCTURE: '3E 2',
+        TYPE_STRUCTURE: 'D',
+      },
+    };
+
+    STRUCTURES.push(STRUCTURES_ELEVE);
+
+  });
+
+  return STRUCTURES;
+};
+
+run();
+
+module.exports = { run };

--- a/high-level-tests/load-testing/package.json
+++ b/high-level-tests/load-testing/package.json
@@ -11,7 +11,8 @@
   "scripts": {
     "preinstall": "npx check-engine",
     "report": "artillery report report/index.json",
-    "start": "artillery run --config config/common.yml -e localhost -o report/index.json scenarios/signup-and-placement.yml"
+    "start": "artillery run --config config/common.yml -e localhost -o report/index.json scenarios/signup-and-placement.yml",
+    "generate-bulk-data:schooling-registrations": "node ./data/generate-schooling-registrations.js"
   },
   "license": "ISC",
   "devDependencies": {

--- a/high-level-tests/load-testing/package.json
+++ b/high-level-tests/load-testing/package.json
@@ -16,6 +16,7 @@
   "license": "ISC",
   "devDependencies": {
     "artillery": "^1.6.0-27",
-    "faker": "^4.1.0"
+    "faker": "^4.1.0",
+    "js2xmlparser": "^4.0.1"
   }
 }

